content/_help/trouble-signing-in/issues-with-authentication-methods._en.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: help
+title: Issues with authentication methods
+category: trouble-signing-in
+permalink: /help/trouble-signing-in/issues-with-authentication-methods/
+order: 3
+---
+
+Depending on the authentication methods you’ve set up, you may still be able to access your Login.gov account. After you’re able to sign in, make sure you’ve set up more than one authentication method to avoid losing access to your account. 
+
+<div class="usa-alert usa-alert--warning margin-bottom-4" role="status">
```

Comment:
not needed for this PR, but we've made jekyll partials/components for alerts in other repos, might be useful here so that we don't have to copy-paste as much boilerplate

example: https://github.com/18F/identity-handbook/blob/9a00acd69840c82533a1bdfddd888957d955b11e/_articles/devops-scripts.md?plain=1#L268-L272